airbyte-integrations/connectors/source-github/source_github/source.py

```diff
@@ -198,53 +200,56 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
         page_size = config.get("page_size_for_large_streams", DEFAULT_PAGE_SIZE_FOR_LARGE_STREAM)
 
         organization_args = {"authenticator": authenticator, "organizations": organizations}
-        organization_args_with_start_date = {**organization_args, "start_date": config["start_date"]}
+        organization_args_with_config = {"authenticator": authenticator, "organizations": organizations, "config": config}
+        organization_args_with_start_date_and_config = {**organization_args, "start_date": config["start_date"], "config": config}
         repository_args = {"authenticator": authenticator, "repositories": repositories, "page_size_for_large_streams": page_size}
-        repository_args_with_start_date = {**repository_args, "start_date": config["start_date"]}
+        repository_args_with_config = {"authenticator": authenticator, "repositories": repositories, "page_size_for_large_streams": page_size, "config": config}
+        repository_args_with_start_date_and_config = {**repository_args, "start_date": config["start_date"], "config": config}
```

Comment:
Please don't create new variables to only have a new object. Add the `api_url` to organization_args, repository_args.

Reply:
@btkcodedev are you still thinking to finish this one?